spec/features/projects/show_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.describe 'projects show ', type: :feature do
+  describe 'as a user' do
```

Comment:
I'd change this to `as a visitor`. Not important for this one, but in the future, we'll have different types of users (admin, visitor, etc)